test/mocha/10-api.js

```diff
@@ -114,7 +114,7 @@ describe('bedrock-data-hub-storage', () => {
         database.hash(mockData.docWithUniqueAttributes.id));
       record.doc.should.deep.equal(mockData.docWithUniqueAttributes);
     });
-    it('should fail to insert a document with a unique attribute', async () => {
+    it.skip('should fail to insert a document with a unique attribute', async () => {
```

Comment:
Intentional .skip here?

Reply:
Yes, because `unique` indexes are turned off at the moment.